torchbox/torchbox.com
# Event block

Link to Tickets
- [BE](https://torchbox.monday.com/boards/1192293412/pulses/1396565970)
- [FE](https://torchbox.monday.com/boards/1192293412/pulses/1396387229)
- [Figma link](https://www.figma.com/file/QdMXBHr57mN1svsaNQ7Ggj/Torchbox-2024-(Master)?type=design&node-id=3568-50544&mode=design&t=68Jr42KoH9gM8XEa-4)

### Description of Changes Made
Added an EventBlock block to the body streamfield of HomePage and ServicePage

### How to Test
- Navigate to the home page or any ServicePage page type. In the body streamfield, ensure that block named `Event` exists
- Ensure that the details in the `Event` block are rendered correctly on the page

### Screenshots
<details><summary>Desktop</summary>
<img width="2160" alt="image" src="https://github.com/torchbox/torchbox.com/assets/40872556/9f469ef3-cd2f-4715-9e3a-85a4ed7e1a66">
</details>

<details><summary>Mobile</summary>
<img width="452" alt="image" src="https://github.com/torchbox/torchbox.com/assets/40872556/de926fd7-05cc-4502-a4d9-3fd23827f379">
</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
